src/main/java/com/google/swarm/tokenization/common/DLPTransform.java

```diff
@@ -241,14 +245,34 @@ public void processElement(
 
   static class ConvertInspectResponse
       extends DoFn<KV<String, InspectContentResponse>, KV<String, TableRow>> {
+
+    private String jobName;
+
+    public ConvertInspectResponse(String jobName) {
+      this.jobName = jobName;
+    }
+
     private final Counter numberOfInspectionFindings =
-        Metrics.counter(ConvertInspectResponse.class, "NumberOfInspectionFindings");
+        Metrics.counter(ConvertInspectResponse.class, "numberOfInspectionFindings");
+
+    private final Counter numberOfTimesFindingsTruncated =
```

Comment:
Can we put a comment on these two metrics? What it does and purpose?  @BhupiSindhwani

Reply:
@santhh - I pushed the new commit with the comments in place for the counters